packages/x-charts/src/ChartsLabel/ChartsLabel.tsx

```diff
@@ -20,16 +20,9 @@ const Root = styled('span', {
   name: 'MuiChartsLabel',
   slot: 'Root',
   overridesResolver: (props, styles) => styles.root,
-})<{ ownerState: ChartsLabelProps }>(({ theme }) => ({
-  ...theme.typography.caption,
-  color: (theme.vars || theme).palette.text.primary,
-  lineHeight: undefined,
-  display: 'flex',
-}));
+})<{ ownerState: ChartsLabelProps }>(() => ({}));
```

Comment:
Very very tiny nitpick: It's bothering me a bit that the mark and the label are not perfectly aligned vertically. It's most likely because of the lineHeight of the label 🙈 We could maybe go with a `lineHeight: '100%'` in this case
![image](https://github.com/user-attachments/assets/fbe59994-ba25-4aff-b305-e6d95b6a367f)

Reply:
I think one the issues also is that the `square mark` has an odd size of `13px` 🤔